src/test/kotlin/no/nav/syfo/cronjob/VerifyPartnerIdCronjobSpek.kt

```diff
@@ -0,0 +1,123 @@
+                azureOpenidConfigTokenEndpoint = environment.azureOpenidConfigTokenEndpoint,
+                httpClient = externalMockEnvironment.mockHttpClient,
+            )
+            val partnerinfoClient = PartnerinfoClient(
+                azureAdClient = azureAdClient,
+                syfoPartnerinfoClientId = environment.syfoPartnerinfoClientId,
+                syfoPartnerinfoUrl = environment.syfoPartnerinfoUrl,
+                httpClient = externalMockEnvironment.mockHttpClient,
+            )
+            val behandlerService = BehandlerService(
+                fastlegeClient = FastlegeClient(
+                    azureAdClient = azureAdClient,
+                    fastlegeRestClientId = environment.fastlegeRestClientId,
+                    fastlegeRestUrl = environment.fastlegeRestUrl,
+                    httpClient = externalMockEnvironment.mockHttpClient,
+                ),
+                partnerinfoClient = PartnerinfoClient(
+                    azureAdClient = azureAdClient,
+                    syfoPartnerinfoClientId = environment.syfoPartnerinfoClientId,
+                    syfoPartnerinfoUrl = environment.syfoPartnerinfoUrl,
+                    httpClient = externalMockEnvironment.mockHttpClient,
+                ),
+                database = database,
+            )
+            val cronJob = VerifyPartnerIdCronjob(
+                behandlerService = behandlerService,
+                partnerinfoClient = partnerinfoClient,
+            )
+            beforeEachTest {
+                database.dropData()
+                clearAllMocks()
+            }
+
+            describe("Cronjob sjekker eksisterende behandlerkontor") {
+                it("Cronjob virker selv om ingen kontor") {
+                    runBlocking {
+                        cronJob.verifyPartnerIdJob()
+                    }
+                }
+                it("Cronjob disabler duplikat kontor med udatert partnerID") {
+                    val kontorId1 = database.createKontor(
+                        partnerId = PARTNERID,
+                        herId = OTHER_HERID,
+                        navn = KONTOR_NAVN,
+                    )
+                    val kontorId2 = database.createKontor(
+                        partnerId = OTHER_PARTNERID,
+                        herId = OTHER_HERID,
+                        navn = KONTOR_NAVN,
+                    )
+                    runBlocking {
+                        cronJob.verifyPartnerIdJob()
+                    }
+                    val kontor1 = database.getBehandlerKontorById(kontorId1)
+                    kontor1.dialogmeldingEnabled shouldBeEqualTo null
+                    val kontor2 = database.getBehandlerKontorById(kontorId2)
+                    kontor2.dialogmeldingEnabled shouldNotBe null
+                }
+                it("Cronjob gjør ingenting med duplikat kontor med udatert partnerID hvis allerede disabled") {
```

Comment:
Denne sjekken er jo i praksis helt lik den over? 🤔 Siden det uansett er kontor1 som ville blitt disabled, har det ingen påvirkning på testen at det er lagt på `enabled = false` på linje 93. Vi verifiserer jo heller ingenting annet ved objektet, så det er ingen garanti for at resten av objektet er som før, hvis det er det man ønsker å teste.